src/app/client/src/app/modules/public/module/signup/signup-routing.module.ts

```diff
@@ -10,6 +10,7 @@ const routes: Routes = [
     path: '', component: SignupComponent,
     data: {
       hideHeaderNFooter : true,
+      hideLocatoionPopup: true,
```

Comment:
please add it in other modules as well recover and signup sign-in

Reply:
Now I have added in signup module and I will add in sign in and recover module